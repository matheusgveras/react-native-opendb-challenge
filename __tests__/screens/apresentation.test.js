import 'react-native';
import React from 'react';
import Apresentation from '../../src/screens/apresentation';
import renderer from 'react-test-renderer';
describe('testing screen apresentation', () => {
  it('renders scren apresentation correctly', () => {
    const tree = renderer.create(<Apresentation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});