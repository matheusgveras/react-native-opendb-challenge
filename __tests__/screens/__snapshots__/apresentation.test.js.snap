// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#cccccc",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#444",
          "fontSize": 30,
        }
      }
    >
      Welcome Quiz
    </Text>
    <Text
      style={
        Object {
          "color": "#444",
          "fontSize": 16,
          "marginBottom": 20,
        }
      }
    >
      Project opendb api exemple
    </Text>
    <Image
      source={
        Object {
          "testUri": "../../../src/resources/images/splash.png",
        }
      }
      style={
        Object {
          "height": 300,
          "width": 300,
        }
      }
    />
  </View>
  <View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#00306B",
            "borderRadius": 0,
            "borderTopColor": "#00306B",
            "borderTopWidth": 1,
            "height": 75,
            "justifyContent": "center",
            "opacity": 1,
            "paddingHorizontal": 15,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#00306B",
              "flexDirection": "row",
              "marginHorizontal": 10,
            }
          }
        >
          <Text
            bold={true}
            style={
              Object {
                "color": "#fff",
                "flex": 1,
                "fontWeight": "bold",
                "letterSpacing": 2,
              }
            }
            upper={true}
          >
            Start Quiz
          </Text>
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Object {
                  "color": "#B5B5B5",
                },
                Object {
                  "fontFamily": "Ionicons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`testing screen apresentation renders scren apresentation correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#cccccc",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#444",
          "fontSize": 30,
        }
      }
    >
      Welcome Quiz
    </Text>
    <Text
      style={
        Object {
          "color": "#444",
          "fontSize": 16,
          "marginBottom": 20,
        }
      }
    >
      Project opendb api exemple
    </Text>
    <Image
      source={
        Object {
          "testUri": "../../../src/resources/images/splash.png",
        }
      }
      style={
        Object {
          "height": 300,
          "width": 300,
        }
      }
    />
  </View>
  <View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#00306B",
            "borderRadius": 0,
            "borderTopColor": "#00306B",
            "borderTopWidth": 1,
            "height": 75,
            "justifyContent": "center",
            "opacity": 1,
            "paddingHorizontal": 15,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#00306B",
              "flexDirection": "row",
              "marginHorizontal": 10,
            }
          }
        >
          <Text
            bold={true}
            style={
              Object {
                "color": "#fff",
                "flex": 1,
                "fontWeight": "bold",
                "letterSpacing": 2,
              }
            }
            upper={true}
          >
            Start Quiz
          </Text>
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Object {
                  "color": "#B5B5B5",
                },
                Object {
                  "fontFamily": "Ionicons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
